### PR TITLE
added nix flake directions

### PR DESCRIPTION
**Pull request by IllustratedMan-code**
### Changes
- Adds a little note to the README that describes how to run the flake with `file` as an example command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ pacman -S gum
 
 # Nix
 nix-env -iA nixpkgs.gum
+# Or try the latest version
+nix run "github:charmbracelet/gum" -- file
```
Comment (maaslalani) [inline README.md:112]:
```suggestion
nix run "github:charmbracelet/gum" -- --help
```

File: README.md [hunk 2]
```diff
@@ -108,6 +108,8 @@ pacman -S gum
 
 # Nix
 nix-env -iA nixpkgs.gum
+# Or try the latest version
```
Comment (maaslalani) [inline README.md:111]:
```suggestion
# Or, with flakes
```

**Comment by IllustratedMan-code** [comment]
Thanks! I wasn't sure how to phrase it, and help is a much better example option.

**Comment by maaslalani** [comment]
No worries, thanks for the PR! and for keeping the flakes up to date!

**Pull request merged**